src/figuresClasses.ts

```diff
@@ -1,11 +1,73 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
```

Comment:
you can define type Shape with such types
` type Shape = 'triangle' | 'circle' | 'rectangle'`

Reply:
The same is for color